packages/react-component-library/src/components/Timeline/Timeline.tsx

```diff
@@ -63,9 +63,12 @@ function isComponentOf<T extends TimelineComponent>(
 
 function extractChildrenOf<T>(
   children: timelineChildrenType | timelineChildrenType[],
-  names: string[]
+  names: string[],
+  inverse?: boolean
```

Comment:
If this parameter is staying then let's change the function to `extractChildren`